ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/Block.hs

```diff
@@ -73,6 +73,10 @@ mkByronHash :: CC.ABlockOrBoundaryHdr ByteString -> ByronHash
 mkByronHash = ByronHash . CC.abobHdrHash
 
 instance ConvertRawHash ByronBlock where
+  -- TODO if @cardano-ledger@ exposes @ShortByteString@ variants of
+  -- 'CC.hashToBytes' and 'CC.unsafeHashFromBytes', we can use them to
+  -- override 'toShortRawHash' and 'fromShortRawHash' to avoid going through a
+  -- 'ByteString'.
```

Comment:
Byron is already using `ShortByteString`, but it just doesn't expose it. See https://github.com/input-output-hk/cardano-ledger-specs/blob/master/byron/crypto/src/Cardano/Crypto/Hashing.hs

Reply:
Done in https://github.com/input-output-hk/cardano-ledger-specs/pull/1581.